web/package-lock.json

```diff
@@ -42,6 +42,7 @@
         "markdown-it": "^12.0.2",
         "markdown-it-anchor": "^6.0.1",
         "markdown-it-attrs": "^4.1.0",
+        "markdown-it-footnote": "^3.0.3",
```

Comment:
Why are you adding this?

Reply:
The `k-web-theme` submodule is shared among many other websites. So some websites use that footnote feature.